src/EditorFeatures/Core.Wpf/EditorFeatures.Wpf.csproj

```diff
@@ -38,6 +38,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Imaging" Version="$(MicrosoftVisualStudioImagingVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.ImageCatalog" Version="$(MicrosoftVisualStudioImageCatalogVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Text.UI.Wpf" Version="$(MicrosoftVisualStudioTextUIWpfVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="$(MicrosoftVisualStudioThreadingVersion)" />
```

Comment:
Don't add if it's coming from project references.

Reply:
I'll fix this up.  I agree adding this is wrong

Reply:
Let us revert this if it is not needed.

---
In reply to: [192248134](https://github.com/dotnet/roslyn/pull/27027#discussion_r192248134) [](ancestors = 192248134)